ddtrace/tracer: Flaky TestPushPayload

This test appears flaky, see: https://app.circleci.com/pipelines/github/DataDog/dd-trace-go/4034/workflows/ddb5cc10-6b02-4a31-acfc-0fd0852636e3/jobs/28421/tests#failed-test-0

```
Failed
=== RUN   TestPushPayload
    tracer_test.go:1529: timed out in 200ms waiting for 2 trace(s)
--- FAIL: TestPushPayload (1.38s)
```

--- a/ddtrace/tracer/tracer_test.go
+++ b/ddtrace/tracer/tracer_test.go
@@ -65,7 +65,7 @@ func TestMain(m *testing.M) {
 }
 
 func (t *tracer) awaitPayload(tst *testing.T, n int) {
-	timeout := time.After(200 * time.Millisecond * timeMultiplicator)
+	timeout := time.After(time.Second * timeMultiplicator)
 loop:
 	for {
 		select {
@@ -1520,7 +1520,7 @@ func startTestTracer(t interface {
 			tick <- time.Now()
 			return
 		}
-		d := 200 * time.Millisecond * timeMultiplicator
+		d := time.Second * timeMultiplicator
 		expire := time.After(d)
 	loop:
 		for {

